examples/bri-3/test/bri.postman_collection.json

```diff
@@ -0,0 +1,158 @@
+{
+	"info": {
+		"_postman_id": "4d16569f-8f2c-43e5-a86e-b35e0c94efb2",
+		"name": "bri",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "http://localhost:3000/subjects",
+			"request": {
+				"method": "PUT",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n    \"id\": \"a300552d-622c-44d2-962e-3bf5a393429a\",\n    \"name\": \"hello3\",\n    \"desc\": \"world2\",\n    \"publicKey\": \"xyz2\"\n}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "http://localhost:3000/subjects",
+					"protocol": "http",
+					"host": [
+						"localhost"
+					],
+					"port": "3000",
+					"path": [
+						"subjects"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "http://localhost:3000/subjects Copy",
```

Comment:
name contains Copy at the end, below Copy 2, probably by mistake, also in get all no need for `all` in the name should be clear from rest route

Reply:
Fixed